### DISCUSSION TITLE
The following routes are ambiguous

### DISCUSSION THREAD

**Discussion by Etnic** [dotnet/AspNetCore.Docs#25773] [Q&A]
Hello,

I cloned the repo and I would like to run project ..AspNetCore.Docs\aspnetcore\blazor\samples\5.0\BlazorSample_WebAssembly\BlazorSample.csproj. I just build a project and run it. I am receiving in console a lot of error for example: The following routes are ambiguous. I am adding different name for that duplicates route, but is there any way how to fix that problem? I would expect that after cloning repo the project can run without any errors.

![image](https://user-images.githubusercontent.com/80886305/165938507-1e0a1a3e-caf1-468f-ab8e-a78df587ca69.png)

**Comment by Rick-Anderson** [comment]
That's an old project, you should be using https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/blazor/samples/6.0
